_events/2022-0503.markdown

```diff
@@ -0,0 +1,18 @@
+---
+
+eventdate: 2022-05-03T015:00
+tz: UTC -7
+title: OpenSearch Community Meeting - 2022-05-03
+online: true
+signup:
+    url: https://www.meetup.com/OpenSearch/events/285396685
+    title: Join on Meetup
+
+---
+
+Join us for an open office community meeting.
```

Comment:
Maybe you want to indicate the open Q&A in the title?

Reply:
made a suitable change to address this.